extension/ts/extensions/resize.ts

```diff
@@ -0,0 +1,95 @@
+import {ForEachFormElement, onFormRender} from "../entities/bookForm";
+import {debounce} from "../util/debounce";
+import {getLastFormRender} from "../util/lastFormRender";
+
+/**
+ * This file is dedicated to saving the sizes of the text areas in the book form
+ * If you double-click the text area while it is REALLY small, or if you don't
+ * use the website for a day, then it will go back to normal
+ */
+
+const RESIZE_EVENT = "resize";
+const SIZE_LOCAL_STORAGE_KEY = "_resize-data";
+const MIN_HEIGHT = 10;
+const MIN_WIDTH = 40;
+const RESIZE_DEBOUNCE_MS = 10;
+const ONE_DAY_MS = 86400000;
+
+interface SizeRecord extends Size {
+	id: string;
+}
+
+interface Size {
+	width: string;
+	height: string;
+}
+
+interface SizeData {
+	[id: string]: {width: string; height: string};
```

Comment:
this value's type and `Size` look the same, is there a reason we're not doing `[id: string]: SIze`?

Reply:
good catch my b